qt/scientific_interfaces/ISISReflectometry/GUI/Batch/BatchPresenter.h

```diff
@@ -65,6 +65,7 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL BatchPresenter
   void notifySettingsChanged() override;
   void anyBatchAutoreductionResumed() override;
   void anyBatchAutoreductionPaused() override;
+  void reductionPaused() override;
```

Comment:
@gemmaguest I know it's unrelated and low priority, but the naming of these functions imply they are getters (i.e. `reductionPaused()` would return a `bool` indicating if the reduction was paused).

A better name might be `notifyReductionPaused()`, etc.